bin/export_grafana_snapshot.py

```diff
@@ -155,17 +156,18 @@ def click_share_link(dashboard_url):
     print("dashboard url:{0}".format(dashboard_url))
     driver.get(dashboard_url)
 
-    element = driver.find_element_by_class_name("navbar-button--share")
-    element.click()
+    share_button = WebDriverWait(driver, 60).until(
+        EC.presence_of_element_located((By.CLASS_NAME, "navbar-button--share")))
+    share_button.click()
 
-    snapshot_tap = WebDriverWait(driver, 1).until(
+    snapshot_tap = WebDriverWait(driver, 15).until(
         EC.presence_of_element_located((By.LINK_TEXT, "Snapshot")))
     snapshot_tap.click()
 
-    snapshot_share = WebDriverWait(driver, 1).until(
+    snapshot_share = WebDriverWait(driver, 60).until(
         EC.presence_of_element_located((By.CLASS_NAME, "btn-secondary")))
     snapshot_share.click()
-    snapshot_link = WebDriverWait(driver, 10).until(
```

Comment:
Can make these values configurable?

Reply:
That might introduce extra confusion. Actually setting the timeout to a higher value just make it safer to get the elements but would not affect the time actually waiting. So should be fine to keep it as fixed value?

Reply:
made the value configurable after offline discussion